stablehlo/dialect/Version.h

```diff
@@ -59,6 +60,14 @@ class Version {
 };
 
 mlir::Diagnostic& operator<<(mlir::Diagnostic& diag, const Version& version);
+llvm::raw_ostream& operator<<(llvm::raw_ostream& os, const Version& version);
+
+template <typename VersionedInterface>
+bool isLegalVersionForTarget(VersionedInterface& interface,
+                             Version const& target) {
```

Comment:
`const Version&` as the more conventional option in our codebase? (Here and everywhere else - this is something that I seem to have missed when reviewing the previous PR).

Reply:
Fixed everywhere. Previous company had the opposite coding standard, to avoid the contingency in the rule: "const applies to the thing left of it. ~If there is nothing on the left then it applies to the thing right of it.~"

Fixed this and all other params to use `const ClassName&`.